src/Definition/RuntimeDefinition.php

```diff
@@ -27,9 +27,7 @@ class RuntimeDefinition implements DefinitionInterface
      */
     public function __construct(?array $explicitClasses = null)
     {
-        if ($explicitClasses) {
-            $this->setExplicitClasses($explicitClasses);
-        }
+        $this->setExplicitClasses($explicitClasses ?? []);
```

Comment:
Maybe this line should be preceeded by `$this->explicitClasses = [];` in case `$explicitClasses` is `null` and remove it from the method `RuntimeDefinition::setExplicitClasses()`?